app/services/import/persist-licence-versions.service.js

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+/**
+ * Persists the licence versions, purposes and conditions
+ * @module PersistLicenceVersionsService
+ */
+
+const LicenceVersionModel = require('../../models/licence-version.model.js')
+const LicenceVersionPurposeModel = require('../../models/licence-version-purpose.model.js')
+const PrimaryPurposeModel = require('../../models/primary-purpose.model.js')
+const SecondaryPurposeModel = require('../../models/secondary-purpose.model.js')
+const PurposeModel = require('../../models/purpose.model.js')
+
+/**
+ * Saves the licence versions, purposes and conditions
+ *
+ * @param {ImportLicenceVersionType[]} licenceVersions
+ * @param {string} licenceId
+ */
+async function go (licenceVersions, licenceId) {
+  return Promise.all(licenceVersions.map(async (version) => {
```

Comment:
I don't think this needs to return anything. 

We can test the DB entires. 

The .returning statement does not seem to limit the response either.

Reply:
remove from here update test.